roles/wildfly_systemd/tasks/main.yml

```diff
@@ -71,6 +71,34 @@
     remote_src: true
     force: no
 
+- block:
```

Comment:
@guidograzioli I'm splitting hair here, but maybe this should lived in a separate file, just to avoid bloating the main.yml. The include_tasks could also be conditional to wildfly_enable_yml_config, removing the need for a block.